commercial/app/controllers/commercial/HostedContentController.scala

```diff
@@ -10,14 +10,28 @@ import views.html.hosted.{episode1Video, guardianHostedPage, teaserVideo}
 object HostedContentController extends Controller {
 
   def renderHostedPage(pageName: String) = Action { implicit request =>
-    val page = HostedPage(pageName)
+    val pageUrl = controllers.commercial.routes.HostedContentController.renderHostedPage(pageName).absoluteURL
```

Comment:
I don't think so.  This is only going to be used for 2 or 3 pages, before we build a capi-based solution.  It gets used a lot in views to resolve URLs.